docs/conf.py

```diff
@@ -58,6 +58,10 @@
     'sphinxcontrib_hdl_diagrams',
 ]
 
+# Uncomment for YosysHQ/fpga-toolchain
```

Comment:
I wonder if this can be passed as a CLI or envvar. In #73, I use `make -C docs html` for building the docs. That's something I do in many other projects with Docs based on Sphinx. Hence, I'd love to do `HDL_DIAGRAM=system:prebuilt make -C docs html` or anything similar, instead of manually modifying the conf script.

Nonetheless, I think we can handle this a separated PR, after this is merged.